src/leases/helpers.js

```diff
@@ -3118,3 +3134,39 @@ export const clearUnsavedChanges = () => {
 export const getLeasesWithContractNumber = (leasesForContractNumbers: LeaseList): boolean => {
   return (get(leasesForContractNumbers, 'count') > 0);
 };
+
+/**
+ * Destructures a nested lease object in a related lease.
+ * @param {Object} lease
```

Comment:
In this typing system is it possible to create a type for lease and use it here?